Annotato/Annotato/Views/Document/Edit/Annotation/AnnotationView.swift

```diff
@@ -88,10 +88,22 @@ class AnnotationView: UIView {
 
     @objc
     private func didPan(_ sender: UIPanGestureRecognizer) {
-        superview?.bringSubviewToFront(self)
+        guard let superview = superview else {
+            return
+        }
+        let currentCenter = viewModel.center
```

Comment:
Would previousCenter sound clearer? Because the center is changed afterwards
```suggestion
        let previousCenter = viewModel.center
```